Modules/Filtering/DisplacementField/include/itkTransformToDisplacementFieldFilter.hxx

```diff
@@ -205,8 +206,13 @@ TransformToDisplacementFieldFilter<TOutputImage, TParametersValueType>::Nonlinea
       // Compute corresponding input pixel position
       transformedPoint = transform->TransformPoint(outputPoint);
 
-      displacement = transformedPoint - outputPoint;
-      outIt.Set(displacement);
+      displacementPoint = transformedPoint - outputPoint;
+      // Cast PointType -> PixelType
+      for (IndexValueType idx = 0; idx < ImageDimension; ++idx)
+      {
+        displacementPixel[idx] = static_cast<PixelType::ValueType>(displacementPoint[idx]);
```

Comment:
Modules/Filtering/DisplacementField/include/itkTransformToDisplacementFieldFilter.hxx:213:46: error: need 'typename' before 'itk::TransformToDisplacementFieldFilter<TOutputImage, TParametersValueType>::PixelType::ValueType' because 'itk::TransformToDisplacementFieldFilter<TOutputImage, TParametersValueType>::PixelType' is a dependent scope